packages/server-ct/package.json

```diff
@@ -2,7 +2,12 @@
   "name": "@packages/server-ct",
   "version": "0.0.0",
   "private": true,
-  "files": [
-    "src"
-  ]
+  "main": "index.js",
+  "scripts": {
+    "test-unit": "ts-mocha --config ./test/.mocharc.js"
+  },
+  "devDependencies": {
+    "@types/mocha": "5.2.7",
```

Comment:
seems they have more up-to-date types: https://www.npmjs.com/package/@types/mocha

Reply:
And I am not even using the ones I installed